README.md

```diff
@@ -109,14 +88,20 @@ Currently, the following Technical Advisory Groups are active:
 * [TAG Observability](https://github.com/cncf/tag-observability)
 * [TAG Environmental Sustainability](https://github.com/cncf/tag-env-sustainability)
 
+The current Technical Advisory Groups would be spun down during 2025 and replace with new Technical Advisory Groups and TOC Sub Projects, listed below.
+
+## New Technical Advisory Groups and TOC Sub Projects
 
-## Working Groups
+### New Technical Advisory Groups
 
-The TOC has created the following working groups to investigate and discuss the following topics:
+* [TAG Developer Experiance](./tags/tag-developer-experience/)
+* [TAG Infrastructure](./tags/tag-infrastructure/)
+* [TAG Operational REsilience](./tags/tag-operational-resilience/)
+* [TAG Security and Compliance](./tags/tag-security-and-compliance/)
+* [TG Workloads-foundation](./tags/tag-workloads-foundation/charter.md)
 
-| Working Group | Chair            | Meeting Time                          | Minutes/Recordings |
-|---------------|------------------|---------------------------------------|--------------------|
-| [CI](https://github.com/cncf/wg-ci)         |   | Inactive | [Youtube](https://www.youtube.com/playlist?list=PLj6h78yzYM2P3_A3ujWHSxOu1IO_bd7Zi)
-| [Serverless](https://github.com/cncf/wg-serverless) | Ken Owens        |  [Thu of every week at 9AM PT](https://zoom.us/my/cncfserverlesswg)                                     | [Youtube](https://www.youtube.com/playlist?list=PLj6h78yzYM2Ph7YoBIgsZNW_RGJvNlFOt)
+### TOC Sub Projects
 
-All meetings are on the public CNCF calendar: https://goo.gl/eyutah
+* [Contributor Strategy Subproject and Advocacy](./toc_subprojects/contributor-strategy-subproject-and-advocacy/)
+* [Mentoring Subproject](./toc_subprojects/mentoring-subproject/)
+* [Project Reviews Subproject](./toc_subprojects/project-reviews-subproject/)
```

Comment:
Please add "Artificial Intelligence Sub Project"

Reply:
Aren't AI initiatives instead of subprojects?

Reply:
Artificial Intelligence Sub Project added, the link will not work as I have to update the tags.yaml in a next PR.